### ISSUE TITLE
Compose events helper

### ISSUE THREAD

**Issue by souporserious** [renatorib/react-powerplug#32]
Would you consider adding a helper utility to help compose events if you want to use the same ones being used by a react-powerplug component?

A contrived example, but I'm using something like this right now:
```jsx
function composeEvents(internalEvents, externalEvents) {
  return Object.keys(externalEvents).reduce((composedEvents, key) => {
    const internal = internalEvents[key]
    const external = externalEvents[key]
    return {
      ...composedEvents,
      [key]: internal
        ? (event, ...args) => {
            internal(event, ...args)
            external(event, ...args)
          }
        : external,
    }
  }, {})
}

// usage
const HoveredDiv = ({ children, onMouseEnter, onMouseLeave, ...restProps }) => (
  <Hover>
    {({ isHover, bindHover }) => (
      <div
        {...composeEvents({ onMouseEnter, onMouseLeave }, bindHover)}
        {...restProps}
        children={children(isHover)}
      />
    )}
  </Hover>
)
```

**Comment by souporserious** [comment]
Love it! Want me to add a PR? I should have time today or tomorrow.

**Comment by renatorib** [comment]
@souporserious Feel free :D

**Comment by renatorib** [comment]
Fixed in #71
